src/main/resources/static/javascript/mainApp/general.controller.js

```diff
@@ -650,8 +650,24 @@
             let reader = new FileReader();
             reader.onload = function (e) {
                 const str = e.target.result;
-                const namesInFile = str.split(/[\r\n,]+/);
-                $scope.addMembers($scope.listName, namesInFile);
+                if (inputFile.type === "text/csv") {
+                    const namesInFile = str.split(/[\r\n]+/);
+                    const firstRow = namesInFile[0].split(",");
+                    const indexOfUhNumber = firstRow.findIndex((header) => header.includes(Message.Csv.UUID_COLUMN_HEADER));
+                    if (indexOfUhNumber < 0) {
+                        $scope.displayDynamicModal(
+                            Message.Title.INVALID_FILE,
+                            `CSV file does not contain ${Message.Csv.UUID_COLUMN_HEADER} in the header`,
+                        );
+                        return;
+                    }
+
+                    const UHNumbersInFile = namesInFile.map((row) => row.split(",")[parseInt(indexOfUhNumber, 10)]).slice(1, -1);
```

Comment:
I see you used parseInt() to get around the Codacy issue. I recommend using Number() instead because it is a little cleaner to read.